.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish
+
+on:
+  push:
+    branches: [ main ]
+    tags: [ 'v*' ]
+
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: publish-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish-crates:
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up context
+        id: project_context
+        uses: FranzDiebold/github-env-vars-action@v2.3.1
+
```

Comment:
Is it actually used?

Reply:
Ah yes, I thought it was necessary for the environment variable but I misunderstood the purpose of this action. Thanks.